test/test_collections.rb

```diff
@@ -180,13 +180,15 @@ class TestCollections < JekyllUnitTest
       @collection = @site.collections["methods"]
     end
 
-    should "include the symlinked file as it resolves to inside site.source" do
-      assert_includes @collection.filtered_entries, "um_hi.md"
-      refute_includes @collection.filtered_entries, "/um_hi.md"
-    end
+    unless Jekyll::Utils::Platforms.windows?
+      should "include the symlinked file as it resolves to inside site.source" do
+        assert_includes @collection.filtered_entries, "um_hi.md"
```

Comment:
You should be able to do `skip "Unsupported test on Windows" if Jekyll::Utils::Platforms.windows?` inside the test.  The reason this is preferable over the swallowing if is it produces inconsistent test numbers, and doesn't let people know why it's that way.